src/EditorFeatures/Core.Cocoa/Structure/StructureTaggerProvider.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.ComponentModel.Composition;
+using Microsoft.CodeAnalysis.Editor.Shared.Utilities;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.Shared.TestHooks;
+using Microsoft.VisualStudio.Text.Editor;
+using Microsoft.VisualStudio.Text.Projection;
+using Microsoft.VisualStudio.Text.Tagging;
+using Microsoft.VisualStudio.Utilities;
+
+namespace Microsoft.CodeAnalysis.Editor.Implementation.Structure
+{
+    [Export(typeof(ITaggerProvider))]
+    [TagType(typeof(IStructureTag))]
+    [ContentType(ContentTypeNames.RoslynContentType)]
+    internal partial class StructureTaggerProvider :
+        AbstractStructureTaggerProvider
+    {
+        [ImportingConstructor]
+        [Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
+        public StructureTaggerProvider(
+            IThreadingContext threadingContext,
+            IForegroundNotificationService notificationService,
+            IEditorOptionsFactoryService editorOptionsFactoryService,
+            IProjectionBufferFactoryService projectionBufferFactoryService,
+            IAsynchronousOperationListenerProvider listenerProvider)
+                : base(threadingContext, notificationService, editorOptionsFactoryService, projectionBufferFactoryService, listenerProvider)
+        {
+        }
+
+        internal override object? GetCollapsedHintForm(StructureTag structureTag)
+        {
+            return CreateElisionBufferForTagTooltip(structureTag).CurrentSnapshot.GetText();
```

Comment:
interesting.  what behavior does this have for VS4Mac users?  Just plain text?

Reply:
This matched the current implementation they had. The "remove unused code" commit is me deleting their forked code for creating a view which was commented out.